INSTALL.CMake

```diff
@@ -80,5 +80,10 @@ cmake -DCMAKE_BUILD_TYPE=Release -DPYTHON=YES -DCMAKE_CXX_FLAGS_RELEASE=" -Wno-r
 make -j 4
 sudo make install
 
-For a special compilation with all the options for fast code, including targeting the machine's processor type, use:
--DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_FLAGS_RELEASE="-O3 -fno-math-errno -funsafe-math-optimizations -fno-rounding-math -fno-signaling-nans -fcx-limited-range -march=native -DNDEBUG"
+For a special compilation with all the options for fast code, including targeting the machine's processor type, use for example:
+-DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_FLAGS_RELEASE="-O3 -fno-math-errno -fno-signaling-nans -msse2 -march=native -DNDEBUG"
```

Comment:
btw, isnt't it equiv. to -Ofast?

Reply:
I do not think: https://gcc.gnu.org/onlinedocs/gcc/Optimize-Options.html#index-Ofast
These are just examples that worked for me.